dotcom-rendering/src/layouts/LiveLayout.tsx

```diff
@@ -357,10 +352,12 @@ export const LiveLayout = (props: WebProps | AppsProps) => {
 
 						<Section
 							fullWidth={true}
-							borderColour={brandLine.primary}
+							borderColour={sourcePalette.brand[600]}
 							showTopBorder={false}
 							padSides={false}
-							backgroundColour={brandBackground.primary}
+							backgroundColour={themePalette(
+								'--article-background',
+							)}
```

Comment:
```suggestion
							backgroundColour={sourcePalette.brand[400]}
```

This will fix the broken subnav